Error with unusual page numbers

I am getting the following error on some citations:

```
[2] 2018-02-01 16:13:26,744 [ERROR] Failed to process BibTeX file c:\users\mtb~1\appdata\local\temp\tmpu53hxk.bib: invalid literal for int() with base 10: 'G66' (io.py:309)
ERRORED: invalid literal for int() with base 10: 'G66'
[7] 2018-02-01 16:13:31,559 [ERROR] Failed to process BibTeX file c:\users\mtb~1\appdata\local\temp\tmpc_9fsl.bib: invalid literal for int() with base 10: 'F48' (io.py:309)
ERRORED: invalid literal for int() with base 10: 'F48'
```
The first one refers to this:  ``` Leonardo Uieda and Valéria C. F. Barbosa (2012). ”Robust 3D gravity gradient inversion by planting anomalous densities.” GEOPHYSICS, 77(4), G55-G66. https://doi.org/10.1190/geo2011-0388.1 ```, while the latter refers to the following: ```Leonardo Uieda, Valéria C. F. Barbosa, and Carla Braitenberg (2016). ”Tesseroids: Forward-modeling gravitational fields in spherical coordinates.” GEOPHYSICS, 81(5), F41-F48. https://doi.org/10.1190/geo2015-0204.1```

It looks like the page is expected to be an integer, which obviously G66 and F48 are not. Is there a sensible way to fix this?

Behaviour seems to be the same regardless of if it is done as a `due.cite(Doi(...))` or as a `due.cite(BibTeX(...))`, just for completeness.

This is to (finally) finish doing this for the [fatiando project](https://github.com/fatiando/fatiando/pull/423), so if there is something that we need to change on our side, as opposed to something in the duecredit python module (I installed the newest one on PyPi yesterday) let me know.

Thanks for the effort you guys have put in so far.

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -32,7 +32,7 @@ classifiers = [
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
 dependencies = [
-    "citeproc-py>=0.4",
+    "citeproc-py>=0.5",
     "looseversion",
     "packaging",
     "requests",

